src/main/java/com/revature/RevRelay/controllers/UserController.java

```diff
@@ -169,4 +169,10 @@
         return ResponseEntity.ok(userService.updateEmail(userID, email.substring(1, email.length() - 1)));
     }
 
+    @PutMapping("/update")
```

Comment:
If the function with the UserUpdateDTO is going to be used instead of the one that uses UserDTO, the existing PutMapping("/current") should be changed to use the UserUpdateDTO model and method instead of having a separate "/update" endpoint.

Reply:
Thank you, Ill implement that